file.go

```diff
@@ -61,6 +61,7 @@ func (f *file) MapEntries() (me []Message)                  { return nil }
 func (f *file) SourceCodeInfo() SourceCodeInfo              { return f.SyntaxSourceCodeInfo() }
 func (f *file) SyntaxSourceCodeInfo() SourceCodeInfo        { return f.syntaxInfo }
 func (f *file) PackageSourceCodeInfo() SourceCodeInfo       { return f.packageInfo }
+func (f *file) Dependents() []Entity                        { return nil }
```

Comment:
should package be the Dependent for this `File` entity?

Reply:
imo, no because packages aren't entities and the file already has a reference to its own package if the end user needed a reference to it. i'm anticipating that dependents for files will just be extensions/file level annotations.